examples/foundry/image_classification.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<p>Should the title use "annotations" or "predictions" ?</p><ol><li>Could we format parameters like this --&gt; <code>predictions_ontology_mapping</code> please?</li><li><code>predictions_ontology_mapping </code> for this one, I think we need to clarify, something like, "A dictionary containing the mapping of the model's ontology feature schema ids to the project's ontology feature schema ids.</li><li><code>override_existing_annotations_rule</code> for this one we are missing one resolution</li><li class="ql-indent-1"><code>OverrideWithAnnotations = "OVERRIDE_WITH_ANNOTATIONS" </code> Do we know what this is ? and how to use it?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Hey! Override with annoations is meant for when you want to override the annotations on a project from one project to another. That is why I did not include since it would not be application for that method. I believe it does the exact same as OverrideWithPredictions when it is with a model. Though it lets me pass in either or and gives the same result. Im sending in some commits now from my changes so far</p>